frontend/src/employee-frontend/components/application-page/ApplicationEditView.tsx

```diff
@@ -718,9 +725,42 @@ export default React.memo(function ApplicationEditView({
               <InputWarning text={i18n.application.preferences.unitMismatch} />
             ) : null}
             {preferredUnits.map((unit, i) => (
-              <HorizontalContainer key={unit.id}>
+              <PreferredUnitGridContainer key={unit.id}>
                 <Link to={`/units/${unit.id}`}>{`${i + 1}. ${unit.name}`}</Link>
-                <Gap size="s" horizontal />
+                <Button
+                  appearance="inline"
+                  icon={faArrowUp}
+                  text={i18n.documentTemplates.templateEditor.moveUp}
```

Comment:
Vaikka dokumenttitemplateista löytyykin jo sopivat tekstit näille nappuloille, niin lisäisin uudet käännösavaimet sopivaan paikkaan `i18n.application` alle näitä tomintoja varten.